src/index.ts

```diff
@@ -146,7 +151,13 @@ export class SupabaseStrategy extends
     const user = await this.getUser(session.access_token)
```

Comment:
@mitchelvanbever do you think we should to the same to reduce getUser calls or its premature optimization ?

Reply:
I don't think we should do it. With the refreshing it results in actual failed requests whereas the getUser shouldn't error if it has a valid token. So I think if we were to apply the same pattern here as for the refreshing it could result in some weird edge cases.